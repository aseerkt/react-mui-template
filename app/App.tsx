import { Route, BrowserRouter, Routes } from 'react-router-dom';
import routerConfig from './utils/routerConfig';

const App = () => {
  return (
    <>
      <BrowserRouter>
        <Routes>
          {routerConfig.map(({ path, component: Component }) => (
            <Route key={path} path={path} element={<Component />} />
          ))}
        </Routes>
      </BrowserRouter>
    </>
  );
};

export default App;
