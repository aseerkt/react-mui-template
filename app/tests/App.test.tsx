import { render } from '@testing-library/react';
import App from '../App';

describe('<App /> tests', () => {
  it('should match snapshot', () => {
    const { baseElement } = render(<App />);
    expect(baseElement).toMatchSnapshot();
  });
});
