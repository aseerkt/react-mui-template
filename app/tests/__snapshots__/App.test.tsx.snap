// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> tests should match snapshot 1`] = `
<body>
  <div>
    <div>
      <h1>
        Mohit Home Page
      </h1>
    </div>
  </div>
</body>
`;
